src/rest/meetings/meeting_functions.ts

```diff
@@ -112,20 +112,33 @@ export function handleMeetingFetch(roomService: RoomService,
 }
 
 
-function mergeMeetings(roomMeetings: RoomMeetings[], userMeetings: Meeting[]): RoomMeetings[] {
-  const userMeetingsMap = userMeetings.reduce((acc, meeting) => {
-    acc.set(meeting.id, meeting);
-    return acc;
-  }, new Map<string, Meeting>());
+/*
+Can't match by meeting id when using different user perspectives
+Don't match by location since bookings by BookIt and Outlook are different
+ */
+function matchMeeting(meeting: Meeting, userMeetings: Meeting[]) {
```

Comment:
It's totally crazy that we have to do this. I mean, I believe it, but damn Microsoft.